### ISSUE TITLE
Falco Installation with certificates

### ISSUE THREAD

**Issue by xi2817-aajgaonkar** [falcosecurity/charts#387]
**Describe the bug**
Getting bug while installation. I was following this guide https://falco.org/blog/falco-kind-prometheus-grafana/#install-falco and it gave me following error
```
Error: INSTALLATION FAILED: template: falco/templates/daemonset.yaml:14:8: executing "falco/templates/daemonset.yaml" at <include "falco.podTemplate" .>: error calling include: template: falco/templates/pod-template.tpl:13:23: executing "falco.podTemplate" at <include (print $.Template.BasePath "/certs-secret.yaml") .>: error calling include: template: falco/templates/certs-secret.yaml:6:11: executing "falco/templates/certs-secret.yaml" at <include "falco.fullname" .>: error calling include: template: falco/templates/_helpers.tpl:14:14: executing "falco.fullname" at <.Values.fullnameOverride>: nil pointer evaluating interface {}.fullnameOverride
```

**How to reproduce it**
```
helm install falco ./falco     --set-file \
certs.server.key=/tmp/certs/server.key,certs.server.crt=/tmp/certs/server.crt,certs.ca.crt=/tmp/certs/ca.crt    \
 --set falco.grpc.enabled=true,falco.grpcOutput.enabled=true
```

**Expected behaviour**

**Screenshots**

![image](https://user-images.githubusercontent.com/88836921/182339747-15fab758-cdfc-4fdf-9bbd-1a8e98bf13c2.png)

**Environment**

- Falco version:

- System info:

- Cloud provider or hardware configuration:
- OS:
- Windows WSL2 Linux distro

- Kernel:
- WSL 2
- 
![image](https://user-images.githubusercontent.com/88836921/182340055-717977a5-cad7-4af3-9c72-2cf7a3641dff.png)

- Installation method:

**Additional context**

**Comment by alacuku** [comment]
Hi @xi2817-aajgaonkar, thank you for reporting the issue.
I took a quick look and it seems that the problem is here: https://github.com/falcosecurity/charts/blob/c3ea76f4a1556c699fa421116d225cf73f93e907/falco/templates/certs-secret.yaml#L1-L2
We are changing the `scope` using the `with` keyword causing the subsequent `falco.fullname` helper to fail. 
Hopefully today I will open PR with the fix.

**Comment by leogr** [comment]
Fixed by #388 
/close
